### PR TITLE
[docs] Add link to GitHub issues under Troubleshooting

### PR DESCRIPTION
**Pull request by sarah11918**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

Removing the banner re: active development also removed the only direct link to GitHub issues. I've added this back in the Troubleshooting section as a helpful directive for people wanting to know whether there is an open issue related to their problem.

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Great work @sarah11918, LGTM ✅

File: docs/src/content/docs/getting-started.mdx [hunk 1]
```diff
@@ -126,4 +126,4 @@ See the growing list of guides in the sidebar for help adding content and custom
 
 If your answer cannot be found in these docs, please visit the [full Astro Docs](https://docs.astro.build) for complete Astro documentation. Your question may be answered by understanding how Astro works in general, underneath this Starlight theme.
 
-You can also get help in the [Astro Discord](https://astro.build/chat/) from our active, friendly community! You can post questions in our `#support` forum or visit our dedicated `#starlight` channel to discuss current development and more!
+You can also check for any known [Starlight issues on GitHub](https://github.com/withastro/starlight/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), and get help in the [Astro Discord](https://astro.build/chat/) from our active, friendly community! Post questions in our `#support` forum with the "starlight" tag, or visit our dedicated `#starlight` channel to discuss current development and more!
```
Comment (TheOtterlord) [inline docs/src/content/docs/getting-started.mdx:129]: is the query intended or just from the github extension?

Comment (sarah11918) [reply docs/src/content/docs/getting-started.mdx:129]: It's from the Github default link, and I didn't hate it enough to change it. :smile: But, not opposed to just the straight up link!

Comment (sarah11918) [reply docs/src/content/docs/getting-started.mdx:129]: @TheOtterlord , propose the change for credit!

**Review by delucis — APPROVED** [review]
LGTM! The query is the default link Refined GitHub generates. Happy for @sarah11918 and @TheOtterlord to decide what’s best here 🫡

File: docs/src/content/docs/getting-started.mdx (hunk 1 above)
Comment (TheOtterlord) [inline docs/src/content/docs/getting-started.mdx:129]:
```suggestion
You can also check for any known [Starlight issues on GitHub](https://github.com/withastro/starlight/issues), and get help in the [Astro Discord](https://astro.build/chat/) from our active, friendly community! Post questions in our `#support` forum with the "starlight" tag, or visit our dedicated `#starlight` channel to discuss current development and more!
```

**Pull request merged**